### ISSUE TITLE
The static-routes file doesn't exist if it is a docker installation

### ISSUE THREAD

**Issue by jorikfon** [mikopbx/Core#701]
И проверить как применяется в обычных установках https://t.me/mikopbx_dev/14741

**Comment by boffart** [comment]
Оно и не должно в docker работать
У нас вообще MikoPBX сетью не управляет в контейнере, это должно настраиваться на хостсистеме.

**Comment by jorikfon** [comment]
Да, верно. Но есть момент, после редактирования файла маршрут не появился в обычной установке, может оно требует ребут?

**Comment by jorikfon** [comment]
Уже смоделировал, поправил. Не трогай ;)
